docs/sources/basics/exemplars.md

```diff
@@ -0,0 +1,21 @@
++++
+title = "Exemplars"
+description = "Exemplars"
+keywords = ["grafana", "concepts", "exemplars", "prometheus"]
+weight = 400
++++
+
+# Exemplars
+
+An exemplar is a specific trace representative of a repeated pattern of data in a given time interval. It helps you identify higher cardinality metadata from specific events within time series data.
+
+Suppose your company’s website is experiencing a surge in traffic volumes. While more than eighty percent of the users are able to access the website in under two seconds, some users are experiencing a higher than normal response time resulting in bad user experience. To identify the factors that are contributing to the latency, you have to compare a trace for a fast response against a trace for a slow response. Given the vast amount of data in a typical production environment, it will be extremely laborious and time consuming effort. 
```

Comment:
```suggestion
Suppose your company website is experiencing a surge in traffic volumes. While more than eighty percent of the users are able to access the website in under two seconds, some users are experiencing a higher than normal response time resulting in bad user experience

To identify the factors that are contributing to the latency, you must compare a trace for a fast response against a trace for a slow response. Given the vast amount of data in a typical production environment, it will be extremely laborious and time-consuming effort. 
```